pom.xml

```diff
@@ -138,8 +138,28 @@
 		<!-- property to skip surefire tests during failsafe execution -->
 		<!--suppress UnresolvedMavenProperty -->
 		<skip.surefire.tests>${skipTests}</skip.surefire.tests>
+		<opentelemetry-semconv.version>1.23.1-alpha</opentelemetry-semconv.version>
 	</properties>
 
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>io.opentelemetry</groupId>
+				<artifactId>opentelemetry-bom</artifactId>
+				<version>${opentelemetry.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+			<dependency>
+				<groupId>io.opentelemetry</groupId>
+				<artifactId>opentelemetry-bom-alpha</artifactId>
+				<version>${opentelemetry.version}-alpha</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
```

Comment:
I am not for using bom. Overwriting dependencies for many reasons (need for a specific version, CVE fixes and so on) would be a nightmare by using a bom. A bom could override everything itself. To override you should include other boms (where the override could be done) and so on. Tbh I would just use the same way we have been using without boms. @scholzj what's your take on this?

Reply:
I'm not sure I understand the objection, a dependency with a version tag in it will override what ever is imported from the BOM. 

The BOM represents the open telemetry projects tested set of dependencies, thus reducing the  reducing the potential for inter module dependency issues. 

I'd also note that looking at the set of dependencies managed by the Otel bom are not all at the same version (the majority are at the version specified by the bom import but not all)

Reply:
Its also worth mentioning that otel themselves say we [should use](https://arc.net/l/quote/uuvuaxoz) the bom.

Reply:
What if I have to override a transient OTel dependency here in the bridge? And maybe this dependency is coming from a bom imported by the OTel bom itself? I will end to import this other bom here to override the dependency. I still think we should avoid using bom. It has been working fine until today without using boms and with any issues. We have the freedom to change/override dependencies when needed, without any issues.

Reply:
BOM screw up other (transitive) dependencies in a weird ways and make it hard to override them. I would definitely avoid it as well.

Reply:
I've never noticed/realised I was having issues caused by using bom's. I'll remove them.

Reply:
bom's removed in https://github.com/strimzi/strimzi-kafka-bridge/pull/867/commits/25df0af1df7f7ee1ff530e3a91e376f612c895cc